src/modules/main/template-details/fork-template/ForkTemplate.tsx

```diff
@@ -39,9 +39,9 @@ const ForkTemplate = ({ template, onClose }: ForkTemplate.Props): JSX.Element =>
     try {
       await forkTemplate({ templateId: template.id });
 
-      setState({ error: false, pending: false, success: true });
+      setState({ error: '', pending: false, success: true });
     } catch {
-      setState({ error: true, pending: false, success: false });
+      setState({ error: 'Error', pending: false, success: false });
```

Comment:
Change this to Error occured and use this error variable instead of current hard coded Error occured in component code